vcd_cli/network.py

```diff
@@ -624,6 +630,35 @@ def add_subnet_external_network(ctx, name, gateway_ip, netmask, ip_range,
         stderr(e, ctx)
 
 
+@external.command(
+    'enable-subnet',
+    short_help='Enable subnet of an external network.')
+@click.pass_context
+@click.argument('name', metavar='<name>', required=True)
+@click.option(
+    '-g',
+    '--gateway',
+    'gateway_ip',
+    required=True,
+    metavar='<ip>',
+    help='gateway ip of the subnet')
+@click.option('--enable/--disable',
+              'is_enabled',
+              default=None,
+              help='enable/disable the subnet')
+def enable_subnet_external_network(ctx, name, gateway_ip, is_enabled):
+    try:
+        extnet_obj = _get_ext_net_obj(ctx, name)
+
+        ext_net = extnet_obj.enable_subnet(gateway_ip=gateway_ip,
+                                           is_enabled=is_enabled)
+
+        stdout(ext_net['{' + NSMAP['vcloud'] + '}Tasks'].Task[0], ctx)
+        stdout('subnet is enabled successfully.', ctx)
```

Comment:
please modify the message, if user is disabling the subnet then it will still print "subnet is enabled successfully"